projects/every-voice/src/lib/every-voice/every-voice.component.html

```diff
@@ -0,0 +1,3 @@
+<button mat-icon-button (click)="onClick()">
+  <mat-icon>{{ icon }}</mat-icon>
```

Comment:
Should this button's visual state change when synthesis is progressing or playing? This is to prevent multiple clicks by the user because it will appear to be doing nothing.  Use the button to stop play
```suggestion
  <mat-icon>{{ isPlaying? stopIcon : icon }}</mat-icon>
```

Reply:
good idea. done.